sqs_lambda/index.ts

```diff
@@ -48,13 +91,15 @@ export async function handlerFn(event: SQSEvent) {
   await Promise.all(
```

Comment:
We should use `Promise.allSettled` here.  `Promise.all` could leave a set of events partially resolved.

Reply:
Agreed to move this to the next PR if it's needing tsconfig changes.